### ISSUE TITLE
Training description

### ISSUE THREAD

**Issue by Spritea** [bwittmann/syn-cerebral-octa-seg#1]
Thanks for the great work!

I noticed that there are training related files, like `syn_cerebral_octa_seg/trainer.py` and `syn_cerebral_octa_seg/trainer.py`, but related training description, i.e., how to use these files to train the 3D segmentation model with the synthetic dataset is missing.

I want to re-implement the training process and results, could you please provide guidance on how to use these training files to train the 3D segmentation model? Thank you!

**Comment by bwittmann** [comment]
Dear @Spritea,

Thanks for reaching out. 
With this message, I also respond to  #2.

> ... related training description, i.e., how to use these files to train the 3D segmentation model with the synthetic dataset is missing.

We provide our exact training script and instructions right here: https://github.com/bwittmann/syn-cerebral-octa-seg?tab=readme-ov-file#-erase-the-need-for-manual-annotations-by-training-on-synthetic-images. Using our provided script `train.py`, you should be able to exactly reproduce our model's performance. All our parameters are described in `config.yaml`, as stated in the readme. 

> Since the model trained with the synthetic dataset is not available yet, which hinders a direct evaluation on custom data, could you please consider releasing the segmentation model trained by you, thank you!

If you closely follow the instructions provided in our readme, you should be able to train the segmentation model from scratch - this should be the exact same checkpoint that we used for our evaluation. **However, please note that the main point of our work is not to provide a segmentation model, but an adjustable synthesis pipeline that should be adapted to the data at hand first by fine-tuning the parameters of the synthesis pipeline** (see https://github.com/bwittmann/syn-cerebral-octa-seg?tab=readme-ov-file#-create-your-own-customized-synthetic-cerebral-3d-octa-images). 

In summary, I suggest: 
1) adjust the parameters of the synthesis pipeline so that the synthetic data matches the characteristics of your dataset.
2) train the segmentation model on the generated data using our provided scripts.

I hope this helps! Feel free to reach out again if you still have some open questions.
Best, Bastian

**Comment by Spritea** [comment]
Thanks very much for your detailed response! I will check and try it.
